mobile-app/app/screens/WalletNavigator/screens/components/OnboardingCarousel.tsx

```diff
@@ -62,6 +63,14 @@ export function InitialSlide (): JSX.Element {
       >
         {translate('screens/OnboardingCarousel', 'Native DeFi for Bitcoin')}
       </ThemedText>
+
+      <ThemedText
```

Comment:
Can we add a border as shown in mock design?

Reply:
done

Reply:
thanks, consider refactoring into a re-usable component since it has the same UI in two screens?

Reply:
good idea, done